test/functional/signature/version_sig_check.bats

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -L -n test-bundle1 -f /file_1 "$TEST_NAME"
+       	create_version -p "$TEST_NAME" 20 10
+        update_bundle "$TEST_NAME" test-bundle1 --add file_3
+
+}
+
+@test "SIG019: Verify signature for latest_version" {
+	
+	# Verify the latest_version signature though openssl
+	run sudo sh -c "openssl smime -verify -in $WEBDIR/version/latest_version.sig -inform DER -content $WEBDIR/version/latest_version -purpose any -CAfile $FUNC_DIR/Swupd_Root.pem"
```

Comment:
why are you running the openssl command within the "run" helper? This helper is supposed to be used only to run the SUT (system under test). You are not testing openssl, you can (and should) just run this command as any other command in bash

Reply:
Understood, maybe these checks that I have for verification can be turned to bash commands as well.

Reply:
Actually I dont need them ..will remove this